src/components/controls/gamepad-controls.js

```diff
@@ -0,0 +1,110 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
Either way on this –

Pro of removing:

- Gamepad buttons are important, but probably too open-ended to live in core, and not related to position/rotation. My current solution (not in this PR) is firing two events: `gamepadbuttondown` and `gamepadbuttondown:BUTTON_NAME`. So that's an opinionated choice.

Cons of removing:

- Every Oculus owner has a gamepad.